src/plugins/WordCounter.ts

```diff
@@ -93,7 +93,7 @@ export class WordCounter extends LobbyPlugin {
     const topIndex = this.periods.reduce((p, a) => a.index < p ? a.index : p, 1000000);
     // 時間切れのサンプルが溜まってきたら捨てる
     if (this.samples.length / 2 < topIndex && this.samples.length > 100) {
-      this.logger.trace(`gc start len:${this.samples.length}, idx:${topIndex}`);
+      this.logger.trace(`GC start length: ${this.samples.length}, IDX: ${topIndex}`);
```

Comment:
This should be `Samples length` but I left it as is because I currently have a faint idea what those abbreviations mean.
IDX for index?

Reply:
- GC
  - I meant Garbage Collection.
- IDX
  - idx(topIndex) represents the number of the last element included in the measurement period in `this.samples`.
This part is to reduce the number of forward truncations of the array.

Wordcounter plugin is a remnant from the time when the silent criteria for chat was unknown.
Since [the criteria](https://osu.ppy.sh/wiki/en/Bot_account) were already known, this plugin should have been removed.